code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -240,18 +240,17 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 
 	return copy
 
-
-/datum/gas_mixture/copy_from(datum/gas_mixture/sample)
+/datum/gas_mixture/copy_from(datum/gas_mixture/sample, partial = 1)

```

Comment:
Why is this defined twice, here and in 135?

Reply:
Who know. in 135 it has comments about what it do. Comments to all gas_mixture procs.

Reply:
Mix them both into a single proc.
DM will ignore the first definition, which will never be triggered, but it can end up confusing contributors.